java/spacewalk-java.changes

```diff
@@ -1,3 +1,6 @@
+- Update tomcat jars to version 9.0.75 (for dev and test)
```

Comment:
I am not sure we want to mention `(for dev and test)` on the changelogs? But if you have for other branches, keep it this way.

Reply:
same idea.